scripts/tests/dns-failsafe.sh

```diff
@@ -1,27 +0,0 @@
-#!/usr/bin/env bash
-
-# If we set the DNS control to `systemd-resolved` but that's not available,
-# the Client should bail out with an error message and non-zero exit code.
```

Comment:
I don't think this test is up to date with this comment. `systemd-resolved` is now the default and we pretty much rely on it. I also don't see anything in here that stops `systemd-resolved`.